config/read.go

```diff
@@ -533,27 +534,39 @@ func Read(logger *util.Logger, filename string) (Config, error) {
 		}
 	} else {
 		if os.Getenv("DYNO") != "" && os.Getenv("PORT") != "" {
+			pg_uri_regex, _ := regexp.Compile("^postgres(?:ql)?://.*")
```

Comment:
Maybe we could turn this into a package-level `.MustCompile`?

Reply:
Also, it doesn't matter in this case, but in terms of semantics, we should use `\A` (beginning of text) instead of `^` (beginning of line).

Reply:
👍